schema/fair.js

```diff
@@ -38,6 +40,20 @@ const FairType = new GraphQLObjectType({
         return `/${id}`;
       },
     },
+    image: Image,
+    location: {
```

Comment:
I mean, it doesn't make sense, but if its not performative for gravity to return locations in the short JSON, then I guess we handle here.